app/api-v1/services/recipeService.js

```diff
@@ -1 +1,15 @@
-module.exports = {}
+const { addRecipe } = require('../../db')
+
+async function createRecipe(reqBody) {
+  if (!reqBody) {
+    return null
+  }
+
+  const recipe = await addRecipe(reqBody)
```

Comment:
does it need to be asyn? lets return a promise only?


return addReciope(reqBody), also can the body validation be added (if it's not a duplication) `addRecipe` method so this can be redundant

it also feel like we are duplicating on db models e.g. addRecipe / createRecipe

Reply:
admittedly I did not give them the same name, I can change one to match if you prefer.